<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>100</y>
     <width>311</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
self.label.setAlignment(Qt.AlignCenter)
</string>
   </property>
   <property name="text">
    <string>First-Fit Algorithm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>190</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
self.label.setAlignment(Qt.AlignCenter)</string>
   </property>
   <property name="text">
    <string>Welcome!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>310</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(5, 189, 255);
border-radius :20px;
border: 2px solid rgb(3, 104, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton:hover {
	background-color:rgb(3, 104, 255)
}</string>
   </property>
   <property name="text">
    <string>Visualization</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>590</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(5, 189, 255);
border-radius :20px;
border: 2px solid rgb(3, 104, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton:hover {
	background-color:rgb(3, 104, 255)
}</string>
   </property>
   <property name="text">
    <string>Special Cases</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>450</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(5, 189, 255);
border-radius :20px;
border: 2px solid rgb(3, 104, 255);
font: 10pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton:hover {
	background-color:rgb(3, 104, 255)
}</string>
   </property>
   <property name="text">
    <string>Average Performance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>730</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgb(255, 0, 0);
border-radius :20px;
border: 2px solid rgb(180, 0, 0);
font: 10pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton:hover {
	background-color:rgb(180, 0, 0)
}</string>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>29293612.jpg</pixmap>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_5</zorder>
 </widget>
 <tabstops>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_5</tabstop>
 </tabstops>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
